### ISSUE TITLE
Agent deployment failed: Unable to determine HKEY_USERS

### ISSUE THREAD

**Issue by lnx09** [Ylianst/MeshCentral#1792]
Hi,
Since the latest versions of MeshCentral, I am no longer able to deploy the Windows agent through a startup script (GPO) or a deployment tool (OCS Inventory NG).
In a command prompt as SYSTEM, the installer fails to query HKEY_USERS and gets stuck:

```
c:\Users\John.Doe\Downloads>whoami
autorite nt\système

c:\Users\John.Doe\Downloads>meshagent64-GROUP.exe -fullinstall
...Checking for previous installation [NONE]
...Installing serviceUnable to determine HKEY_USERS key for: Système
```
(Windows 10 FR)
No problem with the agent versions from mid-June 2020.

Any help would be appreciated.

**Comment by Ylianst** [comment]
Oh, this is not good. Bryan will need to look into this.

**Comment by Ylianst** [comment]
Bryan is looking into this one. It seems the problem is caused by the agent being installed from the "system" account. Bryan says it should be a quick fix.

**Comment by Ylianst** [comment]
FYI. Bryan fixed this issue and it's in GitHub now. I am holing a bit to get a few unrelated things added to the MeshAgent and will release and update of MeshCentral that includes new agents.

**Comment by Ylianst** [comment]
Just published MeshCentral v0.6.33 with all new MeshAgents that include this fix. Let me know if it works.

**Comment by lnx09** [comment]
Everything works fine with the new agent (GPO, deployment tool, prompt as SYSTEM).
Many thanks for your help.
